### PR TITLE
Announcements

### PR DESCRIPTION
**Pull request by Mtessier809**
# [Cleanup ] | [Fixed typos and moved data in constants]
# [New Feature] | [added toggle effect for followers and all tab on announcements page ]
> Team member: @Mtessier809 
Fixes #97 #98 

## Expected Behavior
Fixed typo issues and moved data into following.js and all.js file in the constants folder.  Following.js contains all the data for the followers tab, and all.js contains data for All tab. Also renamed the Home page directory to Announcements

I also added the toggling effect for the followers tab and all tab. Shows different data depending on which button the user presses. 

Here is the Followers tab 
followers

Here is the All tab
all

### CODE REVIEW

File: frontend/VolunteerOne/components/EventCard/CardBody.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  if (type == "event") {
+    return (
+      <Block>
+        <Text style={styles.eventName}>{data["name"]}</Text>
+        <Text>
+          {data["subject"]}
+          {"\n\n"}
+          {data["description"]}
+          {"\n\n"}
+          When: {data["date"]} {"\n\n"}
```
Comment (cerasamson) [inline frontend/VolunteerOne/components/EventCard/CardBody.js:28]: In the test app, When/Where currently displays even when there's no data. Is it possible to only display these two lines when data["date"] / data["location"] are not empty strings?
```suggestion
          {data["date"] != "" ? <Text> When: {data["date"]} {"\n\n"} </Text> : <Text>{"\n\n"}</Text> }
```

File: frontend/VolunteerOne/components/EventCard/CardBody.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          Where: {data["location"]} {"\n"}
```
Comment (cerasamson) [inline frontend/VolunteerOne/components/EventCard/CardBody.js:29]: Same as above
```suggestion
{data["location"] != "" ? <Text> When: {data["location"]} {"\n\n"} </Text> : <Text>{"\n\n"}</Text> }```

File: frontend/VolunteerOne/components/EventCard/CardHeader.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import { StyleSheet } from "react-native";
+import { Avatar } from "@rneui/themed";
```
Comment (cerasamson) [inline frontend/VolunteerOne/components/EventCard/CardHeader.js:2]: Let's sync on which package to use for the Avatar component, I'm currently using `@react-native-material/core`

**Review by cerasamson — DISMISSED** [review]

File: frontend/VolunteerOne/components/EventCard/CardBody.js (hunk 1 above)
Comment (Mtessier809) [reply frontend/VolunteerOne/components/EventCard/CardBody.js:28]: Yeah we could do that. I just have it hard-coded like that for now so we can see what it looks like. I can add that change in the future

**Review by carmen-lee — APPROVED** [review]
Looks great! Thanks

**Review by mattacy — APPROVED** [review]
Looks good

**Pull request merged**